manifests/j/Joplin/Joplin/2.9.17/Joplin.Joplin.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=QUSU.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Joplin.Joplin
+PackageVersion: 2.9.17
+PackageLocale: en-US
+Publisher: Laurent Cozic
+PublisherUrl: https://github.com/laurent22/joplin
+PublisherSupportUrl: https://github.com/laurent22/joplin/issues
+PrivacyUrl: https://github.com/laurent22/joplin#privacy
+Author: Laurent Cozic
+PackageName: Joplin
+PackageUrl: https://joplinapp.org
+License: MIT License
+LicenseUrl: https://raw.githubusercontent.com/laurent22/joplin/dev/LICENSE
+Copyright: Copyright (c) 2016-2021 Laurent Cozic
```

```suggestion
Copyright: Copyright (c) 2016-2022 Laurent Cozic
```
Copyright was updated in https://raw.githubusercontent.com/laurent22/joplin/dev/LICENSE